package.json

```
@@ -1,15 +1,15 @@
 {
   "name": "inline-scripts",
-  "version": "1.7.0",
+  "version": "1.7.1",
```

```suggestion
  "version": "1.7.0",
```

We update npm version as a standalone commit and by running `npm version 1.7.1`, as opposed to manually editing package.json's version. This way, npm automatically creates a tag in git history and updates package.lock as well.

---

Changed back to 1.7.0